src/libretro/LibretroTranslator.cpp

```diff
@@ -334,171 +309,145 @@ const char* LibretroTranslator::GetKeyName(uint32_t code)
 {
   switch (code)
   {
-    case 0x0008: return "backspace";
```

Comment:
Rename `LibretroTranslator::GetKeyName(uint32_t code)` to `GetLibretroKeyName()` and add the function in this commit as `LibretroTranslator::GetKeyName(XBMCVKey key)`